docs/custom-metric-filter.md

```diff
@@ -83,11 +83,16 @@ in Config Sync.
     ```
```

Comment:
Please mention that in 1.14.0, the output of `kubectl get cm otel-collector-googlecloud -n config-management-monitoring -o yaml` has formatting issue.

Reply:
Done, added note before the command to separate 1.14.0